src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -39,7 +39,7 @@ public Settings(string serviceName = null)
             Port = SettingsReader<int>.Read("Port", 33333);
             DatabaseMaintenancePort = SettingsReader<int>.Read("DatabaseMaintenancePort", 33334);
             ProcessRetryBatchesFrequency = TimeSpan.FromSeconds(30);
-            TimeToRestartAfterCriticalFailure = TimeSpan.FromSeconds(60);
+            TimeToRestartErrorIngestionAfterFailure = TimeSpan.FromSeconds(60);
```

Comment:
Why are we setting this twice? Shouldn't the default be applied with `GetTimeToRestartErrorIngestionAfterFailure`

Reply:
Correct. In fact we already do. Fixed